docs/UserGuide.md

```diff
@@ -404,9 +410,9 @@ before and after 2021-03-31 18:00
 
 #### 4.2.5 Fees
 
-##### Getting monthly fee for a particular student: `fee`
+##### Calculating monthly fee for a particular student: `fee`
 
-Gets the monthly fee for a particular student for a particular month and year.
+Calculates and retrieves the monthly fee for a particular student for a particular month and year.
```

Comment:
What do you think of rephrasing it to "Calculates and **displays**" ?
Feels to me that "retrieves" does not really explain it well enough here?

Reply:
Ah right, now that I read again, retrieves doesn't sound right in this case, I have changed to "displays" in the latest commit